### PR TITLE
Episode podcast feed

### PR DESCRIPTION
**Pull request by kookster**
Add the episode's podcast feed url to the links in the API response

### CODE REVIEW

File: app/models/episode.rb [hunk 1]
```diff
@@ -210,7 +208,11 @@ def file_size
   def copy_media(force = false)
     enclosures.each { |e| e.copy_media(force) }
     all_contents.each { |c| c.copy_media(force) }
-    images.each{ |i| i.copy_media(force) }
+    images.each { |i| i.copy_media(force) }
+  end
+
+  def podcast_feed_url
```
Comment (kookster) [inline app/models/episode.rb:214]: Lots of prettify changes, but this is the actual code addition

File: app/representers/api/episode_representer.rb [hunk 2]
```diff
@@ -51,37 +52,47 @@ class Api::EpisodeRepresenter < Api::BaseRepresenter
...
   end
 
   link :audio_version do
-    URI.join(cms_root, represented.prx_audio_version_uri).to_s if represented.prx_audio_version_uri
+    if represented.prx_audio_version_uri
+      URI.join(cms_root, represented.prx_audio_version_uri).to_s
+    end
+  end
+
+  link :podcast_feed do
```
Comment (kookster) [inline app/representers/api/episode_representer.rb:93]: Again, soooo pretty, but this is the new link

File: test/representers/api/episode_representer_test.rb [hunk 3]
```diff
@@ -71,17 +79,29 @@
...
-    assert_equal json['images'].first['original_url'], episode.image.original_url
+    assert_equal json['images'].first['original_url'],
+                 episode.image.original_url
   end
 
   it 'has enclosure' do
     assert_equal json['_links']['enclosure']['href'], episode.media_url
   end
 
+  it 'has a podcast-feed' do
```
Comment (kookster) [inline test/representers/api/episode_representer_test.rb:90]: And new tests!

File: test/representers/api/episode_representer_test.rb [hunk 4]
```diff
@@ -71,17 +79,29 @@
...
+    episode.podcast.url = nil
+    assert_equal json['_links']['prx:podcast-feed']['href'], episode.podcast.published_url
+  end
+
   it 'can represent a sad, podcast-less episode' do
     episode.podcast_id = nil
     assert_equal json['guid'], "prx__#{episode.guid}"
     assert_match("//#{episode.guid}/", json['_links']['enclosure']['href'])
     assert_nil json['_links']['prx:podcast']
+    assert_nil json['_links']['prx:podcast-feed']
```
Comment (kookster) [inline test/representers/api/episode_representer_test.rb:105]: Make sure if there is no related podcast, then no podcast-feed either

**Review by cavis — APPROVED** [review]
👍  looks good to me!

**Pull request merged**